deploy.sh

```diff
@@ -75,7 +75,7 @@ echo "Building gh-pages branch for $BRANCH"
 if [[ ! -e "$MANIFEST" ]]; then  npm run build || exit 1; else echo "Already built"; fi
```

Comment:
@lirsacc I think we also need to change the test for if the build is done (currently: check for the lmio.appcache file). Guess it should now be test for ./build/lmio.appcache? Same test is  Also, how can this already be built?